src/Form/Type/FormatterType.php

```diff
@@ -44,39 +41,18 @@ final class FormatterType extends AbstractType
     protected $translator;
 
     /**
-     * @var ConfigManagerInterface
+     * @var CKEditorConfigurationInterface
      */
-    protected $configManager;
-
-    /**
-     * @var PluginManagerInterface
-     */
-    protected $pluginManager;
-
-    /**
-     * @var TemplateManagerInterface
-     */
-    private $templateManager;
-
-    /**
-     * @var ToolbarManagerInterface
-     */
-    private $toolbarManager;
-
+    protected $ckEditorConfiguration;
+    
     public function __construct(
         PoolInterface $pool,
         TranslatorInterface $translator,
-        ConfigManagerInterface $configManager,
```

Comment:
Technically this is a BC break, also the class is final :/

Reply:
Corrected, you reviewed too quickly :)